src/Algebra/Consequences/Setoid.agda

```diff
@@ -25,7 +25,7 @@ open import Relation.Unary using (Pred)
 ------------------------------------------------------------------------
```

Comment:
Please fix the 2 style-guide violations in this file.

Reply:
I hope that I have found this and fixed (but not very sure).

Reply:
No, they're still there. Read the section about indentation.

https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#indentation

Reply:
Now, I have fixed there at least two deviations. 
?

Reply:
No, the changes you've just made were not style guide violations and you've now changed existing code for no reason. The **new code** that you've added still has two style guide violations, of a type that I've discussed with you twice before. In particular look at the section I linked you to in the style guide and the types of your new proof.